### PR TITLE
Add api-wrapper

### LINKED ISSUE #13: Create API wrapper to be able to call backend from frontend

### PR DESCRIPTION
**Pull request by yousefa00**
Added api-wrapper in utils directory. Using a log to test in map component.
Resolves #13

### CODE REVIEW

File: frontend/src/components/mapbox.js [hunk 1]
```diff
@@ -20,11 +21,12 @@ class MapBox extends React.Component {
     this.state = {
       lng: 5,
       lat: 34,
-      zoom: 3.70,
+      zoom: 3.7
     };
   }
 
   componentDidMount() {
+    console.log(getResponseByTractID("0"));
```
Comment (chyku) [inline frontend/src/components/mapbox.js:30]: Add a `// TODO` comment so we know to remove this later!

File: frontend/src/utils/apiWrapper.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import axios from "axios";
+
+export const getResponseByTractID = tract_id => {
+  /**
+   * Given:
+   * tract id in database
+   *
+   * Returns all field partners associated to that id upon success
+   * Returns GET_TRACT_DATA_FAIL upon failure
+   */
```
Comment (chyku) [inline frontend/src/utils/apiWrapper.js]: great commenting!!

File: frontend/src/utils/apiWrapper.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    })
+    .catch(error => {
+      return {
+        type: "GET_TRACT_DATA_FAIL",
+        error
+      };
+    });
+};
+
+export default getResponseByTractID;
```
Comment (chyku) [inline frontend/src/utils/apiWrapper.js]: it's really weird that you had to add this at the end but we'll see if we can fix this later

**Review by chyku — CHANGES_REQUESTED** [review]
just 1 tiny thing but great job!!

File: frontend/src/components/mapbox.js [hunk 4]
```diff
@@ -20,11 +21,13 @@ class MapBox extends React.Component {
     this.state = {
       lng: 5,
       lat: 34,
-      zoom: 3.70,
+      zoom: 3.7
     };
   }
 
   componentDidMount() {
+    // TODO: Replace the log below with a valid tract request
```
Comment (chyku) [inline frontend/src/components/mapbox.js]: nice comment i rlly appreciate this

**Review by chyku — APPROVED** [review]
nice comment i rlly appreciate this

**Review by caiyingshen — APPROVED** [review]
🤓

**Pull request merged**